accounts/templates/accounts/base.html

```diff
@@ -19,7 +19,15 @@ <h1>
 		    {% block content %}
```

Comment:
こちらでしか、suggestionができなかったので、ここでさせてもらいます。
```suggestion
11行目 <title>{% block title %}{% endblock %}</title>
```
としてください。そうしないと、titleは反映されないです。ここは前回のレビュー漏れです、すみません。
また、welcomeのhtmlもbase.htmlを継承させてください。それと、settings.pyでtemplatesディレクトリをプロジェクトディレクトリ直下から参照できるようにしているので、accountsディレクトリの中にhtmlファイルを作るのではなく、プロジェクトディレクトリ直下にtemplatesディレクトリを作成し、その直下にbase.htmlとaccountsディレクトリ、welcomeディレクトリを作成してください。

Reply:
少々お待ちください

Reply:
これってbodyタグの中にtitleを入れないといけないということですか？

Reply:
これってbodyタグの中にtitleをいれるってことですか？

Reply:
11行目を上のように変更してくださいということです。suggestionができなかったので、19行目扱いになってしまっています。

Reply:
なるほど！ありがとうございます

Reply:
訂正しました